external/fv3fit/fv3fit/_shared/jacobian.py

```diff
@@ -2,7 +2,7 @@
 import tensorflow as tf
 from typing import Callable, Mapping, Dict
 
-from .layers.normalization import standard_deviation_all_features
+from ..emulation.layers.normalization import standard_deviation_all_features
```

Comment:
I'd advocate for `fv3fit.keras.jacobian` rather than the `fv3fit.keras._shared.jacobian` namespace. Would be nicer to use the former in a jupyter notebook for example (or in the offline diags).